anatomy/tracks/presentation.md

```diff
@@ -0,0 +1,193 @@
+_work in progress_
+
+# Presentation
```

Comment:
The goal of this document is to help explain how documents end up being presented to the student.

Reply:
I think we'll need a clearer name for this, but I can't think of one off hand.

Reply:
Is _display_ or _student_display_ any clearer?

Reply:
🤷

Reply:
I'm gonna merge this PR and we can come up with a better name later.